clients/drcachesim/simulator/cache_fifo.h

```diff
@@ -52,6 +52,8 @@ class cache_fifo_t : public cache_t {
     access_update(int line_idx, int way) override;
     int
     replace_which_way(int line_idx) override;
+    int
+    get_next_way_to_replace(int block_idx);
```

Comment:
It seems worth a comment explaining why there are 2 functions which seem to do the same thing.  The const will help, but a comment will additionally make it clear.